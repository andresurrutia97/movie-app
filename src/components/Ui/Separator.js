import React from 'react';

const Separator = () => <hr className="mb-5 border-gray-600" />;

export default Separator;
