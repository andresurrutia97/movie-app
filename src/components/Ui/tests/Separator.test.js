import React from 'react';
import renderer from 'react-test-renderer';
import { shallow } from 'enzyme';

import Separator from '../Separator';

describe('<Separator/>', () => {
  const component = <Separator />;
  let wrapper;

  beforeEach(() => {
    wrapper = shallow(component);
  });

  it('should render correctly ', () => {
    const tree = renderer.create(component).toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('should contain an <hr/> tag', () => {
    expect(wrapper.find('hr')).toHaveLength(1);
  });

  it('should have className prop equal to "mb-5 border-gray-600"', () => {
    expect(wrapper.props().className).toBe('mb-5 border-gray-600');
  });
});
