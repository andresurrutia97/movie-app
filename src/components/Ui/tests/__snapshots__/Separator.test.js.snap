// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Separator/> should render correctly  1`] = `
<hr
  className="mb-5 border-gray-600"
/>
`;
